public/fonts/vscode_operator_mono_lig-master/README.md

```
@@ -0,0 +1,45 @@
+# Operator Mono w/Ligature Support for VSCode
+
+This repository is meant to help you setup Operator Mono in VSCode. I went through many various guides to get this working so decided to put this repo up to help others interested in getting this working. I have only tested this on Mac OSX as this is what I'm currently using.
+
+The fonts found in the font directory already include ligature support. Below is a list where I obtained the fonts and instructions to build them.
+
+- Free Operator Mono Font: https://www.cufonfonts.com/font/operator-mono
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)